src/object/object_domain.c

```diff
@@ -6261,9 +6261,22 @@ tp_value_coerce_strict (const DB_VALUE * src, DB_VALUE * dest, const TP_DOMAIN *
 	case DB_TYPE_NCHAR:
 	case DB_TYPE_VARNCHAR:
 	  {
-	    if (tp_atonumeric (src, target) != NO_ERROR)
+	    DB_VALUE temp;
```

Comment:
Trivial:
DB_VALUE doesn't have a constructor. Only the domain of the DB_VALUE is initialized in tp_atonumeric.
It doesn't cause any issues for now. But whenever DB_VALUE is used, db_make_null() should be called just in case.

```suggestion
	    DB_VALUE temp;
	    db_make_null (&temp);
```

Reply:
Initializing by calling db_value_domain_init() internally.
Also, it seems that this code will not affect you.
I'll keep this part as it is for now.

If we're going to modify it, we need to modify the same thing for tp_value_cast_internal() calling tp_atonumeric().
Thank you.